fix for "ImportError: .../lib/python2.7/site-packages/mkl_service-2.0.1-py2.7-linux-x86_64.egg/mkl/_py_mkl_service.so: undefined symbol: MKLMPI_Get_wrappers" on 'import mkl' check
author: Kenneth Hoste (HPC-UGent)
--- mkl-service-2.0.1/mkl/setup.py.orig	2019-04-21 18:53:10.000000000 +0200
+++ mkl-service-2.0.1/mkl/setup.py	2019-05-27 15:22:34.739347539 +0200
@@ -37,7 +37,7 @@
     mkl_info = get_info('mkl')
     mkl_include_dirs = mkl_info.get('include_dirs', [])
     mkl_library_dirs = mkl_info.get('library_dirs', [])
-    mkl_libraries = mkl_info.get('libraries', ['mkl_rt'])
+    mkl_libraries = ['mkl_rt'] #mkl_info.get('libraries', ['mkl_rt'])
 
     defs = []
     if any(['mkl_rt' in li for li in mkl_libraries]):
